cfme/tests/configure/test_access_control.py

```diff
@@ -1020,3 +1021,17 @@ def test_delete_default_tenant(appliance):
             row[0].check()
     with error.handler('Default Tenant "{}" can not be deleted'.format(roottenant.name)):
         view.toolbar.configuration.item_select('Delete selected items', handle_alert=True)
+
+
+def test_copied_user_password_inheritance(appliance, group_collection):
+    """Test to verify that dialog for copied user should appear and password field should be
+    empty"""
+    group_name = 'EvmGroup-user'
+    group = group_collection.instantiate(description=group_name)
+    user = new_user(appliance, group)
+    view = navigate_to(user, 'Details')
+    view.toolbar.configuration.item_select('Copy this User to a new User')
+    view = user.create_view(AddUserView)
+    assert view.password_txt.value == '' and view.password_verify_txt.value == ''
```

Comment:
**REQUIRED**
Please make a proper finalizer for the code below. If the test fails that code will be never executed.